requirements/dev-bookworm-requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
+# This file is autogenerated by pip-compile with python 3.7
```

Comment:
We've seen differences in at least indirect dependencies between Python 3.9 (default on Bullseye) and Python 3.10 (default on Bookworm), so it is best to generate each of these files with the corresponding Python version. (Python 3.10 for this one.)

(I don't know if that may work towards fixing the CI failures.)